client/my-sites/plugins/main.jsx

```diff
@@ -567,12 +251,15 @@ export default React.createClass( {
 		return some( this.props.sites.getSelectedOrAllWithPlugins(), site => site && site.jetpack && site.canUpdateFiles );
 	},
 
-	updateAllPlugins() {
-		PluginsActions.removePluginsNotices( this.state.notices.completed.concat( this.state.notices.errors ) );
-		this.state.plugins.forEach( plugin => {
-			plugin.sites.forEach( site => PluginsActions.updatePlugin( site, site.plugin ) );
-		} );
-		this.recordEvent( 'Clicked Update all Plugins', true );
+	showPluginListPlaceholders( isWpCom ) {
+		let showPlaceholders = isEmpty( this.state.plugins );
```

Comment:
@johnHackworth I'm still confused why both of these can't be `const`. I don't see any reassignment to that variable, so why would it have to be rewritten when using `const`? Am I missing something?